newrelic/resource_newrelic_nrql_alert_condition_test.go

```diff
@@ -10,149 +10,129 @@ import (
 )
 
 func TestAccNewRelicNrqlAlertCondition_Basic(t *testing.T) {
+	resourceName := "newrelic_nrql_alert_condition.foo"
 	rName := acctest.RandString(5)
-	resource.Test(t, resource.TestCase{
+
+	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:     func() { testAccPreCheck(t) },
 		Providers:    testAccProviders,
 		CheckDestroy: testAccCheckNewRelicNrqlAlertConditionDestroy,
 		Steps: []resource.TestStep{
+			// Test: Create
 			{
-				Config: testAccCheckNewRelicNrqlAlertConditionConfig(rName),
+				Config: testAccNewRelicNrqlAlertConditionConfig(rName),
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckNewRelicNrqlAlertConditionExists("newrelic_nrql_alert_condition.foo"),
-					resource.TestCheckResourceAttr(
-						"newrelic_nrql_alert_condition.foo", "name", fmt.Sprintf("tf-test-%s", rName)),
-					resource.TestCheckResourceAttr(
-						"newrelic_nrql_alert_condition.foo", "runbook_url", "https://foo.example.com"),
-					resource.TestCheckResourceAttr(
-						"newrelic_nrql_alert_condition.foo", "enabled", "false"),
-					resource.TestCheckResourceAttr(
-						"newrelic_nrql_alert_condition.foo", "term.#", "1"),
-					resource.TestCheckResourceAttr(
-						"newrelic_nrql_alert_condition.foo", "term.0.duration", "5"),
-					resource.TestCheckResourceAttr(
-						"newrelic_nrql_alert_condition.foo", "term.0.operator", "below"),
-					resource.TestCheckResourceAttr(
-						"newrelic_nrql_alert_condition.foo", "term.0.priority", "critical"),
-					resource.TestCheckResourceAttr(
-						"newrelic_nrql_alert_condition.foo", "term.0.threshold", "0.75"),
-					resource.TestCheckResourceAttr(
-						"newrelic_nrql_alert_condition.foo", "term.0.time_function", "all"),
-					resource.TestCheckResourceAttr(
-						"newrelic_nrql_alert_condition.foo", "nrql.#", "1"),
-					resource.TestCheckResourceAttr(
-						"newrelic_nrql_alert_condition.foo", "nrql.0.query", "SELECT uniqueCount(hostname) FROM ComputeSample"),
-					resource.TestCheckResourceAttr(
-						"newrelic_nrql_alert_condition.foo", "nrql.0.since_value", "20"),
+					testAccCheckNewRelicNrqlAlertConditionExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "name", fmt.Sprintf("tf-test-%s", rName)),
```

Comment:
The individual attribute checks can all be removed from this entire file if we are using import checks as the last test step.